pipes/config.yaml

```diff
@@ -274,6 +281,11 @@ ref_genome_dir: "ref_genome"
 # Can be a remote path prefix (s3://, gs://, etc.). The path should contain lastal.{ext} files
 lastal_ref_db_dir: "lastal_db"
 
+# The directory where the reference selection files should be stored
+# Can be a remote path prefix (s3://, gs://, etc.). The path should contain a refsel.fasta file.
+refsel_ref_db_dir: "refsel_db"
```

Comment:
Why not just be the path to the fasta directly?

Reply:
It's the same interface as for lastal_db .  Right now refsel_db contains just the one fasta file, but in the future the reference selection data can get more complex, so it makes sense to have a directory for it.